gap_fit_module.f95

```diff
@@ -61,46 +63,103 @@ module gap_fit_module
 #endif
 
   type gap_fit
-  !% everything from the command line
-     type(Atoms), dimension(:), allocatable :: at
-     
-     character(len=STRING_LENGTH) :: at_file='', core_ip_args = '', e0_str, local_property0_str, &
-     energy_parameter_name, local_property_parameter_name, force_parameter_name, virial_parameter_name, &
-     stress_parameter_name, hessian_parameter_name, config_type_parameter_name, sigma_parameter_name, &
-     config_type_sigma_string, core_param_file, gp_file, template_file, force_mask_parameter_name
-
-     character(len=10240) :: command_line = ''
-     real(dp), dimension(total_elements) :: e0, local_property0
-     real(dp) :: max_cutoff
-     real(dp), dimension(4) :: default_sigma
-     real(dp) :: default_local_property_sigma
-     real(dp) :: sparse_jitter, e0_offset, hessian_delta
-     integer :: e0_method = E0_ISOLATED
-     logical :: do_core = .false., do_copy_at_file, has_config_type_sigma, sigma_per_atom = .true.
-     logical :: sparsify_only_no_fit = .false.
-     integer :: n_frame, n_coordinate, n_ener, n_force, n_virial, n_hessian, n_local_property, min_save, n_species
-     type(extendable_str) :: quip_string
-     type(Potential) :: core_pot
-
-     type(gpFull) :: my_gp
-     type(gpSparse) :: gp_sp
-
-     type(descriptor), dimension(:), allocatable :: my_descriptor
-     character(len=STRING_LENGTH), dimension(200) :: gap_str
-
-     real(dp), dimension(:), allocatable :: delta, f0, theta_uniform, zeta, unique_hash_tolerance, unique_descriptor_tolerance !, theta
-     real(dp), dimension(:,:), allocatable :: sigma
-     integer, dimension(:), allocatable :: n_sparseX, sparse_method, target_type, n_cross, n_descriptors, species_Z, covariance_type
-     integer, dimension(:,:), allocatable :: config_type_n_sparseX
-     character(len=STRING_LENGTH), dimension(:), allocatable :: theta_file, sparse_file, theta_fac_string, config_type, config_type_n_sparseX_string, print_sparse_index
-     logical, dimension(:), allocatable :: mark_sparse_atoms, add_species, has_theta_fac, has_theta_uniform, has_theta_file, has_zeta
-
-     logical :: sparseX_separate_file
-     logical :: sparse_use_actual_gpcov
-     logical :: has_template_file, has_e0, has_local_property0, has_e0_offset
-
+    !% everything from the command line
```

Comment:
Lots of whitespace/indentation fixes. OK for me, but makes the git history a bit harder to analyse.

Reply:
Oh, I forgot to patch-add the changes here. I can change it if you prefer the whitespace cleanup in a separate commit/later.

Reply:
If you don’t mind that would be good to ease checking and to make it clearer what is functionally changing here

Reply:
Do you use the browser to compare the changes? There is an option to hide whitespace changes:
![image](https://user-images.githubusercontent.com/1591390/132467811-03b01896-20ae-49c9-9a51-4e1556c16df9.png)

Can still split it.

Reply:
Thanks - couldn't see how to do this in the PR code review view

Reply:
Ah, ok, found it. No need to split the commit unless you started doing it already.

Reply:
For @albapa's benefit, here's the diff without whitespace https://github.com/libAtoms/GAP/pull/10/files?diff=unified&w=1